data/schema.yaml

```diff
@@ -365,14 +400,35 @@ components:
             title:
               type: string
               example: "Dramatic capsize in America's Cup yacht"
-            versionId:
+            type:
+              enum:
+                - aresMediaMetadata
               type: string
-              example: 'p01jvmg7'
-            versionType:
-              type: string # probably should be an enum in the future
-              example: 'Original'
-            warnings:
-              type: object
+            versions:
+              type: array
+              items:
+                type: object
+                properties:
+                  availableTerritories:
+                    type: object
+                    properties:
+                      nonUk:
+                        type: boolean
+                        example: true
+                      uk:
+                        type: boolean
+                        example: true
+                  duration:
+                    type: number
+                    example: 191
+                  types:
+                    type: array
+                    example: ['Original']
+                  versionId:
+                    type: string
+                    example: 'p01jvmg7'
+                  warnings:
+                    type: object
```

Comment:
Do we have an example of warnings? As it's an object it will probably have it's own nested properties

Reply:
added!